frontend/src/app/modules/application-dashboard/components/application-job-status/graphite-integration/graphite-integration.component.html

```diff
@@ -83,6 +90,57 @@ <h2>{{ 'frontend.de.iteratec.osm.applicationDashboard.jobStatus.graphiteIntegrat
           <td>{{selectedGraphiteServer.prefix ? selectedGraphiteServer.prefix : "no prefix"}}</td>
         </tr>
       </table>
+
+      <form [formGroup]="createServerForm" *ngIf="showCreateSection && !selectedGraphiteServer" class="graphite-form-group">
+        <table>
+          <tr>
+            <td>{{'frontend.de.iteratec.osm.applicationDashboard.jobStatus.graphiteIntegration.address' | translate}}</td>
+            <td>
+              <input type="url"  class="btn btn-sm btn-default"
```

Comment:
The btn* classes should only be used for buttons. Replace this with a matching and consistent style e.g. form-control.
Same for the following inputs.